policy/modules/system/udev.if

```diff
@@ -503,6 +503,7 @@ interface(`udev_read_runtime_files',`
 	')
 
 	files_search_runtime($1)
+	udev_search_runtime($1)
 	read_files_pattern($1, udev_runtime_t, udev_runtime_t)
```

Comment:
Hmm, because of the `read_files_pattern()` there should already be the equivalent of `udev_search_runtime()` access in this interface.

Reply:
Hmm. Unfortunately I didn't save the AVC for why I added this. I think for now I'll drop it and if I can reproduce it in the future I will look into it further.